### ISSUE TITLE
Unique option not working with templates

### ISSUE THREAD

**Issue by Soullivaneuh** [nelmio/alice#291]
The issue is quite simply to understand but was hard to found.

Basically, if you try to set unique option on a template:

```yml
AppBundle\Entity\Domain:
    domain (template):
        name (unique): <domainName()>
        status: 1
        server: @server_*
        ftpLogin: <userName()>
        mysqlLogin: <userName()>
    domain_special_chars (extends domain):
        name: sßnté-çava.æfr
    domain_{0..500} (extends domain): ~
```

This is not working, producing a doctrine constraint exception.

But, if you move back the `name` property outside the template:

```yml
AppBundle\Entity\Domain:
    domain (template):
        status: 1
        server: @server_*
        ftpLogin: <userName()>
        mysqlLogin: <userName()>
    domain_special_chars (extends domain):
        name: sßnté-çava.æfr
    domain_{0..500} (extends domain):
        name (unique): <domainName()>
```

This works properly.

It seems unique option is simply ignored with templates.

**Comment by theofidry** [comment]
Related to #276 as well I guess.

**Comment by theofidry** [comment]
Closed via #359

**Comment by theofidry** [comment]
@Soullivaneuh could you please check if this has been solved by #359?
